.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: ruby
 sudo: false
 rvm:
- - 2.3.1
+ - 2.5.0
```

Comment:
2.5.1?

Reply:
Yep that probably would have been better ;)